### PR TITLE
UI: redesign DatePicker 

### PR DESCRIPTION
**Pull request by negue**
- [x] extract datepicker & settings as component 
- [x] redesign
- [x] today button
- [x] tomorrow button

### CODE REVIEW

**Comment by Tressley** [comment]
This LGTM. 👍

File: package.json [hunk 1]
```diff
@@ -25,6 +25,8 @@
     "domain-middleware": "~0.1.0",
     "eslint": "^6.8.0",
     "eslint-config-habitrpg": "^6.2.0",
+    "eslint-config-standard": "^14.1.1",
```
Comment (negue) [inline package.json]: I think those were because of the datepicker (needed to link it) - will check and remove if not needed anymore

Comment (paglias) [reply package.json]: thanks! apart from this the code looks good to me

**Review by paglias — APPROVED** [review]
LGTM 👍 I'll let you merge it in case you want to change something

**Pull request merged**